src/main/java/com/paneedah/mwc/wearables/Backpacks.java

```diff
@@ -315,93 +315,93 @@ public static void init(Object mod, CompatibleFmlPreInitializationEvent event) {
+//
+//        BergenBackpack = new ItemStorage.Builder()
+//        .withName("bergen_backpack")
+//        .withSize(36)
+//        .withGuiTextureWidth(256)
+//        .withValidItemPredicate(item -> !(item instanceof Weapon))
+//        .withTab(CreativeTabs.MISC)
+//        .withModel(new com.paneedah.mwc.models.BergenBackpack())
+//        .withModelTextureName("bergen_backpack.png")
+//        .withGuiTextureName("big_backpack.png")
+//        .withCustomEquippedPositioning((player, stack) -> {
+//            GL11.glScalef(0.8f, 0.8f, 0.8f);
+//            GL11.glTranslatef(0f, 0f, 0f);
+//            GL11.glRotatef(0F, 0f, 0f, 1f);
+//        })
+//        .withInventoryPositioning(stack -> {
+//            GL11.glScalef(1.1f, 1.1f, 1.1f);
+//            GL11.glTranslatef(-0.2f, -0.5f, 0f);
+//            GL11.glRotatef(-33F, 0f, 1f, 0f);
+//            GL11.glRotatef(20F, 1f, 0f, 0f);
+//        })
+//        .build(ModernWarfareMod.MOD_CONTEXT);
         
-        G2Gunslinger2Backpack = new ItemStorage.Builder()
-        .withName("g2_gunslinger2_backpack")
-        .withSize(10)
-        .withGuiTextureWidth(256)
-        .withValidItemPredicate(item -> (item instanceof Weapon))
-        .withTab(CreativeTabs.MISC)
-        .withModel(new com.paneedah.mwc.models.G2Gunslinger2Backpack())
-        .withModelTextureName("f4m_terminator_backpack.png")
-        .withGuiTextureName("combat_sustainment_backpack.png")
-        .withCustomEquippedPositioning((player, stack) -> {
-            GL11.glScalef(0.8f, 0.8f, 0.8f);
-            GL11.glTranslatef(0f, 0f, 0f);
-            GL11.glRotatef(0F, 0f, 0f, 1f);
-        })
-        .withInventoryPositioning(stack -> {
-            GL11.glScalef(1.1f, 1.1f, 1.1f);
-            GL11.glTranslatef(-0.2f, -0.5f, 0f);
-            GL11.glRotatef(-33F, 0f, 1f, 0f);
-            GL11.glRotatef(20F, 1f, 0f, 0f);
-        })
-        .build(ModernWarfareMod.MOD_CONTEXT);
+//        G2Gunslinger2Backpack = new ItemStorage.Builder()
+//        .withName("g2_gunslinger2_backpack")
+//        .withSize(10)
+//        .withGuiTextureWidth(256)
+//        .withValidItemPredicate(item -> (item instanceof Weapon))
+//        .withTab(CreativeTabs.MISC)
+//        .withModel(new com.paneedah.mwc.models.G2Gunslinger2Backpack())
+//        .withModelTextureName("f4m_terminator_backpack.png")
+//        .withGuiTextureName("combat_sustainment_backpack.png")
+//        .withCustomEquippedPositioning((player, stack) -> {
+//            GL11.glScalef(0.8f, 0.8f, 0.8f);
+//            GL11.glTranslatef(0f, 0f, 0f);
+//            GL11.glRotatef(0F, 0f, 0f, 1f);
+//        })
+//        .withInventoryPositioning(stack -> {
+//            GL11.glScalef(1.1f, 1.1f, 1.1f);
+//            GL11.glTranslatef(-0.2f, -0.5f, 0f);
+//            GL11.glRotatef(-33F, 0f, 1f, 0f);
+//            GL11.glRotatef(20F, 1f, 0f, 0f);
+//        })
+//        .build(ModernWarfareMod.MOD_CONTEXT);
```

Comment:
I do not remember what these are, do you know if we should remove them with Contagion @Paneedah

Reply:
It's an old backpack which is broken that was under the MISC Default Minecraft Creative Tab